src/main/resources/templates/fragments/topnav.html

```diff
@@ -0,0 +1,8 @@
+<div th:fragment="search-navbar">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <form class="form-inline my-2 my-lg-0">
+            <input class="form-control mr-sm-2" id = "pac-input" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+        </form>
+    </nav>
+</div>
```

Comment:
This is for the reusable nav bar! This way you don't have duplicated code for each html page.